examples/gno.land/p/demo/ownable/ownable.gno

```diff
@@ -4,23 +4,42 @@ import (
 	"std"
 )
 
-// Ownable is meant to be used as a top-level object to make your contract ownable OR
-// being embedded in a Gno object to manage per-object ownership.
-type Ownable struct {
+// Ownable defines an interface that requires a method to get the owner's address
+type Ownable interface {
+	Owner() std.Address
+}
+
+// Transferrable embeds Ownable, indicating that an object can transfer and drop an ownership
+type Transferrable interface {
```

Comment:
```suggestion
type transferrable interface {
```

You can unexport it so people will only use the interfaces and the constructors.